Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -312,7 +322,9 @@ public class XPCServer {
 public protocol NonBlockingServer {
     /// Begins processing requests received by this XPC server.
     func start()
-    /// Use an endpoint to create connections to this server
+    /// Use an endpoint to create connections to this server.
```

Comment:
This verb-phrase doesn't really cover what this property does itself (it talks about what you can do down the line). How about splitting that up, such as:

```suggestion
    /// Export an endpoint for this server, which can later be used to create new connections to this server.
```

Reply:
Indeed, incorporated your feedback.